static/elements/chromedash-activity-log.js

```diff
@@ -128,7 +128,11 @@ export class ChromedashActivityLog extends LitElement {
 
   // Display how long ago the comment was created compared to now.
   formatCommentRelativeDate(comment) {
-    const commentDate = new Date(`${comment.created} UTC`);
+    // format date to "YYYY-MM-DDTHH:mm:ss.sssZ" to represent UTC.
+    if (comment.created) {
+      comment.created = comment.created.replace(' ', 'T');
```

Comment:
It seems odd to modify the passed in object.  I think there might be a case where the value with the "T" in it would subsequently get displayed to the user.  It would be better to use a local variable.

Reply:
I agree with this. updated!